docs/resources/scenarios/analyze-web-downloads.md

```diff
@@ -0,0 +1,195 @@
+---
+title: 'Office Scripts sample scenario: Internet Traffic Analysis'
```

Comment:
Suggest making the title here in the metadata consistent with the H1 wording. i.e., "Internet Traffic Analysis" -> "Analyze web downloads"